src/middlewared/middlewared/plugins/support.py

```diff
@@ -169,14 +169,36 @@ async def fetch_categories(self, token):
 
         return data
 
+    @accepts(Password('token'), Str('query'), roles=['SUPPORT_READ'])
+    @returns(List('similar_issues', items=[Dict(
+        'similar_issue',
+        Str('url'),
+        Str('summary'),
+        additional_attrs=True,
+    )]))
+    async def similar_issues(self, token, query):
+        await self.middleware.call('network.general.will_perform_activity', 'support')
+
+        data = await post(
+            f'https://{ADDRESS}/freenas/api/v1.0/similar_issues',
+            data=json.dumps({
+                'token': token,
+                'query': query,
+            }),
+        )
+
+        if 'error' in data:
+            raise CallError(data['message'], errno.EINVAL)
```

Comment:
Should we raise `ValidationError` here so it can be pretty printed in the UI?

Reply:
Why can't `CallError` be pretty-printed?